caffe2/proto/caffe2.proto

```diff
@@ -184,6 +240,23 @@ message OperatorDef {
   // This is an optional string with no assumed characteristics as
   // operators can be constructed in any language.
   optional string debug_info = 10;
+
+  // the domain of the operator
+  // to support legacy operator libs, we use domain to distinguish which
+  // operator lib we refer to:
+  //   - "CAFFE2" means this uses Caffe2 operator library
```

Comment:
the names are probably going to be lowercase to match Symbol abstraction from the code

Reply:
Sure, changed to lowercase.